Astro 10/25 release notes (#1383)
astronomer/docs: astro/configure-deployment-resources.md

    - Click **Delete** next to the email you want to delete.
    - Click **Yes, Continue**.

## Transfer a Deployment to another Workspace 

> Transferring a Deployment moves all DAGs, task history, connections, API keys, and other Astro configurations to another existing Workspace. Only users belonging to the target Workspace can access the Deployment once it is transferred. You can only transfer a Deployment to a Workspace in which you are a Workspace Admin or Editor. 

Reviewer: ```suggestion
You can transfer a Deployment from one Workspace to another as long as the Workspaces are in the same Organization and Astro cluster. You might do this if your organization wants to change which group of users has access to the Deployment.

Transferring a Deployment moves all DAGs, task history, connections, API keys, and other Astro configurations. Running tasks are not be affected or interrupted and tasks will continue to be scheduled.

Only users belonging to the target Workspace can access the Deployment once it is transferred. This means that users with access to the Deployment in the current Workspace could lose access. To transfer a Deployment, you must be a Workspace Admin in the original Workspace and a Workspace Admin or Editor in the target Workspace.
```

    - Click **Delete** next to the email you want to delete.
    - Click **Yes, Continue**.

> ## Transfer a Deployment to another Workspace 
> 
> You can transfer a Deployment from one Workspace to another as long as the Workspaces are in the same Organization and Astro cluster. You might do this if your organization wants to change which group of users has access to the Deployment.
> 
> Transferring a Deployment moves all DAGs, task history, connections, API keys, and other Astro configurations. Running tasks are not be affected or interrupted and tasks will continue to be scheduled.
> 
> Only users belonging to the target Workspace can access the Deployment once it is transferred. To transfer a Deployment, you must be a Workspace Admin in the original Workspace and a Workspace Admin or Editor in the target Workspace.
> 
> 1. In the Cloud UI, select a Workspace and then select a Deployment.
> 2. Click the **Options** menu of the Deployment you want to transfer, and select **Transfer Deployment**. 
> 
>     ![Transfer Deployment in options menu](/img/docs/transfer-deployment.png)
> 
> 3. Select the target Workspace where you want to transfer the Deployment. 
> 4. Click **Transfer Deployment**.
> 

Author: ```suggestion
```

    - Click **Delete** next to the email you want to delete.
    - Click **Yes, Continue**.

## Transfer a Deployment to another Workspace 

Transferring a Deployment moves all DAGs, task history, connections, API keys, and other Astro configurations to another existing Workspace. Only users belonging to the target Workspace can access the Deployment once it is transferred. You can only transfer a Deployment to a Workspace in which you are a Workspace Admin or Editor. 

1. In the Cloud UI, select a Workspace.

> 2. Click the **Options** menu of the Deployment you want to transfer, and select **Transfer Deployment**.

Reviewer 2: ```suggestion
2. Click the **Options** menu of the Deployment you want to transfer, and select **Transfer Deployment**.
```
This is tricky. The Options menu isn't visible until you hover over the Deployment.

![image](https://user-images.githubusercontent.com/104205257/197789789-3f713542-7758-4819-94ae-4046b6078e16.png)

![image](https://user-images.githubusercontent.com/104205257/197790045-32e197a0-48f0-4a78-ace7-4553cc873302.png)

Is there a reason why these controls are obfuscated in this way? If this can't be fixed, perhaps some guidance should be provided? For example:

Hold the pointer over the Deployment you want to transfer, click the **Options** menu, and then select **Transfer Deployment**.

Alternatively, you could instruct users to select a Deployment first.

> 3. Choose the Workspace to which you want to transfer the Deployment. 

Reviewer 2: ```suggestion
3. Select the target Workspace where you want to transfer the Deployment. 
```
Using the language in the UI for consistency.
